log4j-core/src/main/java/org/apache/logging/log4j/core/appender/SyslogAppender.java

```diff
@@ -211,6 +214,10 @@ public String getExceptionPattern() {
             return loggerFields;
         }
 
+		public String getTimestampPrecision() {
```

Comment:
This indentation looks messed up. Spaces, no tabs, could be the issue?

Reply:
Yeah.. issue due to tabs. Was due to problem with my eclipse formatter setting. I have fixed it with latest commit.